runtime/calamari/src/weights/frame_system.rs

```diff
@@ -57,80 +57,84 @@ pub trait WeightInfo {
 /// Weights for frame_system using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for SubstrateWeight<T> {
-    fn remark(_b: u32, ) -> Weight {
-        (11_758_000 as Weight)
+    fn remark(b: u32, ) -> Weight {
+        (9_730_000 as Weight)
+            // Standard Error: 0
+            .saturating_add((1_000 as Weight).saturating_mul(b as Weight))
     }
     fn remark_with_event(b: u32, ) -> Weight {
-        (715_000 as Weight)
+        (2_101_000 as Weight)
```

Comment:
Sometimes, there's always a big increased number.

Reply:
yeah, the remark weight is so low that somehow it often has these weird large swings.
It's remark, it's small - i'll continue to ignore it anytime it happens.